packages/@functionless/validate/test/__snapshots__/validate.test.ts.snap

```diff
@@ -1,968 +1,962 @@
 
 https://functionless.org/docs/error-codes#stepfunctions-calls-to-eventbus-putevents-must-use-object-literals
 
-[7m245[0m   await bus.putEvents({ ...event });
+[7m240[0m   await bus.putEvents({ ...event });
 [7m   [0m [91m                      ~~~~~~~~~~~~[0m
-[96mtest/test-files/step-function.ts[0m:[93m250[0m:[93m23[0m - [91merror[0m[90m Functionless(10023): [0mStepFunctions calls to EventBus putEvents must use object literals
+[96mtest/test-files/step-function.ts[0m:[93m246[0m:[93m23[0m - [91merror[0m[90m Functionless(10023): [0mStepFunctions calls to EventBus putEvents must use object literals
 
 https://functionless.org/docs/error-codes#stepfunctions-calls-to-eventbus-putevents-must-use-object-literals
 
-[7m250[0m   await bus.putEvents({ [source]: "", detail: {}, "detail-type": "" });
+[7m246[0m   await bus.putEvents({ [source]: "", detail: {}, "detail-type": "" });
 [7m   [0m [91m                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~[0m
-[96mtest/test-files/step-function.ts[0m:[93m256[0m:[93m51[0m - [91merror[0m[90m Functionless(10023): [0mStepFunctions calls to EventBus putEvents must use object literals
+[96mtest/test-files/step-function.ts[0m:[93m252[0m:[93m51[0m - [91merror[0m[90m Functionless(10023): [0mStepFunctions calls to EventBus putEvents must use object literals
 
 https://functionless.org/docs/error-codes#stepfunctions-calls-to-eventbus-putevents-must-use-object-literals
 
-[7m256[0m     { source: "", detail: {}, "detail-type": "" },
+[7m252[0m     { source: "", detail: {}, "detail-type": "" },
 [7m   [0m [91m                                                  [0m
-[7m257[0m     { [source]: "", detail: {}, "detail-type": "" }
+[7m253[0m     { [source]: "", detail: {}, "detail-type": "" }
 [7m   [0m [91m~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~[0m
-[96mtest/test-files/step-function.ts[0m:[93m274[0m:[93m15[0m - [91merror[0m[90m Functionless(10024): [0mStepFunctions error cause must be a constant
+[96mtest/test-files/step-function.ts[0m:[93m270[0m:[93m15[0m - [91merror[0m[90m Functionless(10024): [0mStepFunctions error cause must be a constant
 
 https://functionless.org/docs/error-codes#stepfunctions-error-cause-must-be-a-constant
 
-[7m274[0m   throw Error(input.arg);
+[7m270[0m   throw Error(input.arg);
 [7m   [0m [91m              ~~~~~~~~~[0m
-[96mtest/test-files/step-function.ts[0m:[93m302[0m:[93m28[0m - [91merror[0m[90m Functionless(10012): [0mExpected the first argument of $AWS.Lambda.Invoke to be an object literal.
-
-https://functionless.org/docs/error-codes#expected-an-object-literal
-
-[7m302[0m   await $AWS.Lambda.Invoke(event);
-[7m   [0m [91m                           ~~~~~[0m
-[96mtest/test-files/step-function.ts[0m:[93m328[0m:[93m8[0m - [91merror[0m[90m Functionless(10019): [0mUnable to find reference out of application function
```

Comment:
regression?

Reply:
Fixed - it was a regression caused by the removal of `makeIntegration`